lib/shoulda/matchers/independent/delegate_method_matcher.rb

```diff
@@ -188,6 +189,10 @@ def description
             string << " passing arguments #{delegated_arguments.inspect}"
           end
 
+          if delegate_object_is_nil_allowed
+            string << " allowing nil"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.